### DISCUSSION TITLE
jekyll/update/ready-for-action/

### DISCUSSION THREAD

**Discussion by giscus** [OpenGammaProject/Gamma-Spectrum-Database#6] [Announcements]
# jekyll/update/ready-for-action/

Most of the work has been done for now. Today I added the last couple of isotopes that I had lying around to the database and finished up all the remaining pages.

https://gammadb.nuclearphoenix.xyz/jekyll/update/ready-for-action/

**Comment by NuclearPhoenixx** [comment]
Hello World!

**Reply by NuclearPhoenixx** [reply]
Hello again!
